library/alloc/src/rc.rs

```diff
@@ -386,7 +386,35 @@ impl<T> Rc<T> {
     /// returns, calling [`upgrade`] on the weak reference inside `data_fn` will
     /// fail and result in a `None` value.
     ///
+    /// # Why using this method?
+    ///
+    /// It allows to construct a `T` which holds a `Weak<T>` pointer to itself.
+    /// Without this method, you would need to use [`Weak::dangling`] and then
```

Comment:
I don't find the `Weak::dangling` function in the doc. Does this mean `Weak::new`?

Reply:
It should have been `Weak::new` indeed.